tools/test-framework/src/types/wallet.rs

```diff
@@ -33,9 +33,10 @@ pub struct Wallet {
     /// The address for receiving tokens for this wallet.
     pub address: WalletAddress,
 
-    /// The wallet key information in the form of [`KeyEntry`] that
-    /// is used by the relayer.
-    pub key: KeyEntry,
+    // TODO: Parameterize this type on `SigningKeyPair`
+    /// The wallet key information in the form of [`SigningKeyPair`]
+    /// that is used by the relayer.
+    pub key: Secp256k1KeyPair,
```

Comment:
Not that we should eventually do this if we want to test against Solana in the integration tests, but this can be left for a follow-up PR or when full support for Solana is implemented.

Reply:
Sounds good, going to resolve this for now.